### ISSUE TITLE
Allow setting of msgid and msgid domain

### ISSUE THREAD

**Issue by mybooc** [sabuhish/fastapi-mail#143]
1. Extra 'msgid domain' ConnectionConfig parameter

Set your own domain instead of the currently used local hostname.

MAIL_MSGID_DOMAIN: Optional[str] = None

2. Extra 'msgid domain' MailMsg parameter

Dynamically set your own 'msgid domain' together with the MailMsg parameters.

:param msgid_domain

Overrides MAIL_MSGID_DOMAIN parameter

3. Extra 'msgid' MailMsg parameter

Dynamically set the 'msgid' together with the MailMsg parameters.

:param msgid

**Comment by sabuhish** [comment]
I should do research about it and then bring some results, if you have already approach implemented, please feel free to submit PR.

**Comment by LuckyLub** [comment]
@mybooc, it this pull request related #204?

**Comment by mybooc** [comment]
No, this issue is about setting (part of) the Message-ID of an email, a unique message identifier. If you do not supply it, the Message-ID is generated by the SMTP server:
The github email from you had the Message-ID:

Message-ID: <sabuhish/fastapi-mail/issues/143/1775233261@github.com>

**Comment by LuckyLub** [comment]
Ah yes, I see. I thought maybe it was related because of the domain.
